shared/pages/import/oauth.html

```diff
@@ -14,7 +14,7 @@
     <script type="text/javascript" src="/shared/pages/import/js/flow.js"></script>
     <script type="text/javascript" src="/shared/pages/import/js/oauth20.js"></script>
     <script type="text/javascript" src="js/parameters.js"></script>
-    <script type="text/javascript" src="/shared/pages/import/js/curtain.js"></script>
+    <script type="text/javascript" src="/shared/pages/import/js/curtain_panel.js"></script>
     <script type="text/javascript" src="/shared/pages/import/js/oauth_frame.js"></script>
```

Comment:
We need to load `LazyLoader`, otherwise `curtain_panel.js` will not work.

`<script type="text/javascript" src="/shared/js/lazy_loader.js"></script>`